src/Config/Config.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PharIo\ComposerDistributor\Config;
+
+use PharIo\ComposerDistributor\FileList;
+
+class Config
+{
+	/** @var string */
+	private $package;
+
+	/** @var \SplFileInfo|null */
+	private $keyDirectory;
+
+	/** @var \PharIo\ComposerDistributor\FileList */
+	private $phars;
+
+	public function __construct(string $package, ?\SplFileInfo $keyDir, FileList $phars)
```

Comment:
Sollten wir die Reihenfolge der Parameter nicht umdrehen dann?

Reply:
And default $keyDir to null?

Reply:
Done